gitpod-init.sh

```diff
@@ -1,5 +1,10 @@
 #!/usr/bin/env bash
 
+gp env GEM_HOME=/workspace/.rvm
```

Comment:
`gp env` allows you to set environment variables for the Gitpod workspace, but you need to run `eval $(gp env -e)` to make them available in the terminal session.

This is the init script which is why I set them and load them here.